components/system_notice/notices.jsx

```diff
@@ -21,7 +21,7 @@ export default [
         title: (
             <FormattedHTMLMessage
                 id='system_notice.title'
-                defaultMessage='<strong>System Message</strong> from Mattermost'
+                defaultMessage='<strong>Notice</strong> from Mattermost'
```

Comment:
Should "Mattermost" be the SiteName setting?

Reply:
Actually that's a really good question. I checked and we do have "Mattermost" hard-coded all over the place so I'm guessing it's OK. @jasonblais thoughts?

Reply:
Are there examples of where "Mattermost" is hard-coded? If admins haven't seen "Mattermost", Site Name might make sense here.

Shift probably wouldn't want any strings to be hard-coded for whitelabeling purposes.

Reply:
Open the translations file (i18n/en.json) and search for "Mattermost", there's a bunch of hits

Reply:
Ah, they're hardcoded but in en.json. Then it won't be a problem for Shift.

Also, let's leave it as is. 0/5 if we should use the SiteName more often, but this is more consistent with other strings the admin sees (About Mattermost dialog, System Console, ..)

Reply:
Yeah I guess "hard-translated" might have been a better phrase :P Anyways sounds good